doc/user_manual/PostProcessors/Validation.tex

```diff
@@ -133,18 +146,71 @@ \subsubsection{Validation PostProcessors}
   ...
   <Models>
     ...
+    <PostProcessor name="pp1" subType="PPDSS">
+      <Features>outMC1|x1</Features>
+      <Targets>outMC2|x2</Targets>
+      <Metric class="Metrics" type="Metric">dss</Metric>
+      <pivotParameterFeature>time1</pivotParameterFeature>
+      <pivotParameterTarget>time2</pivotParameterTarget>
+      <scale>DataSynthesis</scale>
+      <scaleBeta>1</scaleBeta>
+      <scaleOmega>1</scaleOmega>
+    </PostProcessor>
     <PostProcessor name="pp2" subType="PPDSS">
-      <Features>outMC1|x1,outMC1|y1</Features>
-      <Targets>outMC2|x2,outMC2|y2</Targets>
+      <Features>outMC1|x1</Features>
+      <Targets>outMC2|x2</Targets>
+      <Metric class="Metrics" type="Metric">dss</Metric>
+      <pivotParameterFeature>time1</pivotParameterFeature>
+      <pivotParameterTarget>time2</pivotParameterTarget>
+      <separateFeatureData type="ratio">0.0|0.5</separateFeatureData>
+      <separateTargetData type="ratio">0.0|0.5</separateTargetData>
+      <scale>DataSynthesis</scale>
+      <scaleBeta>1</scaleBeta>
+      <scaleOmega>1</scaleOmega>
+    </PostProcessor>
+    <PostProcessor name="pp3" subType="PPDSS">
+      <Features>outMC1|x1</Features>
+      <Targets>outMC2|x2</Targets>
       <Metric class="Metrics" type="Metric">dss</Metric>
       <pivotParameterFeature>time1</pivotParameterFeature>
       <pivotParameterTarget>time2</pivotParameterTarget>
+      <separateFeatureData type="raw_values">0.2475|0.495</separateFeatureData>
+      <separateTargetData type="raw_values">0.3475|0.695</separateTargetData>
       <scale>DataSynthesis</scale>
-      <scaleBeta>1,1</scaleBeta>
-      <scaleOmega>1,1</scaleOmega>
+      <scaleBeta>1</scaleBeta>
+      <scaleOmega>1</scaleOmega>
     </PostProcessor>
     ...
   <Models>
   ...
+  <DataObjects>
+    ...
+    <HistorySet name="pp1_out">
+      <Output>
+          dss_x2_x1,total_distance_x2_x1,process_time_x2_x1,standard_deviation_x2_x1
```

Comment:
Could you document the outputs of the PPDSS in the user manual? The information that you can get from the PPDSS, and the way to retrieve the information.

Reply:
Added all information of possible postprocessed data that can be retrieved.